templates/products/includes/_products.html.twig

```diff
@@ -34,7 +34,7 @@
 
     <div class="container-fluid product">
         <div class="card_product_container row justify-content-center">
-            {% if products == [] %}
+            {% if products.items == [] %}
```

Comment:
pas beau ça... 
déjà tu es sur que c'est products.items ? (les 2 au pluriel ?)
et ensuite doit y avoir un if is empty plutot qui doit fonctionner dans twig